pkg/cli/cmds/agent.go

```diff
@@ -155,13 +155,13 @@ var (
 	}
 	VPNAuth = &cli.StringFlag{
 		Name:        "vpn-auth",
-		Usage:       "(agent/networking) (experimental) Credentials for the VPN provider. It must include the provider name and join key in the format name=<vpn-provider>,joinKey=<key>",
+		Usage:       "(agent/networking) (experimental) Credentials for the VPN provider. It must include the provider name and join key in the format name=<vpn-provider>,joinKey=<key>[,vpnServerURL=<url>]",
```

Comment:
Nit: don't stutter `vpn` in the serverURL key, we already know this is vpn-related as it's in the vpn-auth flag.
```suggestion
		Usage:       "(agent/networking) (experimental) Credentials for the VPN provider. It must include the provider name and join key in the format name=<vpn-provider>,joinKey=<key>[,serverURL=<url>]",
```

Reply:
See https://github.com/k3s-io/k3s/pull/7807#issuecomment-1606992126

`serverURL` is a bit confusing for Tailscale, because it may refer to exit node (server), for example. `vpnServerURL` indeed stutters, but points more strongly to the VPN "provider", IMO.

This is why initially I proposed `controlURL`, because it's unambiguous and doesn't stutter.

Reply:
Yeah, maybe `controlServerURL`? headscale refers to itself as `implementation of the Tailscale control server` so I think that might be the correct term for what this is?

Reply:
The thing is that I would prefer to be as generic as possible (in case we integrate with other VPN providers). ControlURL, to me, is too Tailscale specific. Zerotier apparently calls them network controllers and Nebula calls them lighthouses. I think `ControlServerURL` seems like a good compromise to me

Reply:
Changed to `controlServerURL`, as suggested.